Makefile

```diff
@@ -308,14 +308,44 @@ undeploy: kustomize ## Undeploy controller from the K8s cluster specified in ~/.
 	done
 	@echo "Namespace 'openshift-dpu-operator' has been removed."
 
```

Comment:
I like the idea of having a way to build with centosstream instead. the tmp files in-tree wont work on my setup. Can you use temp files like we already do in the makefile elsewhere? I will review the PR more deeply soon

Reply:
could you elaborate why that does not work?

Reply:
- The repository on the server is synchronized with my local directory. While it's building, it gets cleaned up
- In addition, in the CI, the whole directory gets removed every time we build, and this slows down because the cache isn't kept across builds